style/_widgetboard.scss

```
@@ -0,0 +1,328 @@
+  clip: rect(0, 0, 0, 0);
+  width: 1px;
+  height: 1px;
+  margin: -1px;
+}
+
+// memo.css
+.widgetboard {
+  padding: 18px;
+
+  background-color: #f4f6fa;
+  border-radius: 8px;
+  box-shadow: 0 0 0 1px #e3e5e8, 0 1px 2px 0 rgba(0, 0, 0, 0.04);
+}
+
+.widgetboard__title {
+  color: #080808;
+  font-size: 1.5rem;
+  font-weight: 800;
+  line-height: 24px;
+}
+
+.widgetboard__memo {
+  position: relative;
+
+  margin-top: 12px;
+
+  background-color: #fff;
+  border-radius: 8px;
+  box-shadow: 0 0 0 1px #ebebeb, 0 1px 2px 0 rgba(0, 0, 0, 0.04);
+}
+
+.memo__header {
+  position: relative;
+
+  display: flex;
+
+  padding: 7px 9px;
+
+  border-radius: 8px 8px 0 0;
+
+  color: #fff;
+  // font-size: 1.5rem;
+
+  background-image: linear-gradient(to top, #00b869, #00c772);
+
+  // bottom dotted
+  &::after {
+    content: "";
+
+    position: absolute;
+    right: 0;
+    bottom: 2px;
+    left: 0;
+
+    height: 1px;
+
+    background-image: linear-gradient(
+      to right,
+      rgba(255, 255, 255, 0.4) 70%,
+      transparent 30%
+    );
+    background-size: 6px 1px;
+  }
+
+  a {
+    padding-left: 10px;
+
+    flex-basis: 50%;
+
+    font-weight: 800;
+    line-height: 34px;
+  }
+}
+
+.memo__lock {
+  margin-left: auto;
+
+  button {
+    position: relative;
+    width: 38px;
+    height: 34px;
+
+    // unlock icon
+    i {
+      position: relative;
+
+      display: inline-block;
+
+      width: 28px;
+      height: 28px;
+
+      border-radius: 50%;
+
+      background-color: #00b266;
+
+      &::before {
+        content: "";
+        position: absolute;
+        top: 0;
+        left: 0;
+        right: 0;
+        bottom: 0;
+
+        width: 20px;
+        height: 20px;
+
+        margin: auto;
+        background-image: url(../img/sp_main.89f0d213.png);
+        background-size: 434px 422px;
+        background-position: -315px -368px;
+        background-repeat: no-repeat;
+      }
+    }
+
+    // tooltip
+    .memo__tooltip {
+      display: none;
+
+      position: absolute;
+      top: 38px;
+      left: 50%;
+
+      padding: 0 16px;
+
+      border: 1px solid #03c75a;
+      border-radius: 20px;
+
+      color: #101010;
+      // font-size: 1.3rem;
+      font-size: 13px;
+      font-weight: 500;
+      line-height: 38px;
+      white-space: nowrap;
+
+      background-color: #fff;
+      box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.15);
+
+      transform: translateX(-50%);
+
+      &::after {
+        content: "";
+
+        position: absolute;
+        left: 50%;
+        bottom: 38px;
+
+        width: 20px;
+        height: 11px;
+
+        background-image: url(../img/sp_main.89f0d213.png);
+        background-repeat: no-repeat;
+        background-size: 434px 422px;
+        background-position: -124px -154px;
+
+        transform: translateX(-50%) rotate(180deg);
+      }
+    }
+
+    &:hover {
+      i {
+        background-color: #009e5a;
+      }
+      .memo__tooltip {
+        display: block;
+      }
```

BEM 컨벤션으로 바꾸면서 누락된거같아요. :)

```suggestion
     .memo__lock-button-tooltip {
        display: block
      }